toolkit/tools/osmodifierapi/os.go

```diff
@@ -13,10 +13,12 @@ import (
 
 // OS defines how each system present on the image is supposed to be configured.
 type OS struct {
-	Hostname string                     `yaml:"hostname"`
-	SELinux  imagecustomizerapi.SELinux `yaml:"selinux"`
-	Users    []imagecustomizerapi.User  `yaml:"users"`
-	Overlays *[]Overlay                 `yaml:"overlays"`
+	Hostname   string                     `yaml:"hostname"`
+	SELinux    imagecustomizerapi.SELinux `yaml:"selinux"`
+	Users      []imagecustomizerapi.User  `yaml:"users"`
+	Overlays   *[]Overlay                 `yaml:"overlays"`
+	Verity     *imagecustomizerapi.Verity `yaml:"verity"`
```

Comment:
The way you are using the fields under `imagecustomizerapi.Verity` is very different from how they are used in MIC. Therefore, you should create a new verity API type that is designed for EMU.

Reply:
fixed in the pr in new repo